src/utils/dateUtil.jsx

```diff
@@ -9,12 +9,12 @@ const createOneDay = days => {
   };
 };
 
-const useDate = () => {
+const generateDates = () => {
```

Comment:
pls keep export consistant 

you don't use default exports in other utils, so export function with named export

or viceversa

Reply:
Now all of them export default. 😎